src/test/groovy/de/eddyson/tapestry/react/integration/AlertDemoSpec.groovy

```diff
@@ -24,7 +24,9 @@ class AlertDemoSpec extends JettyGebSpec {
     then:
     // TODO: this will need a custom ApplicationStatePersistenceStrategy
     // applicationStateManager.get(AlertStorage).alerts.size() == 1
-    helloWorld.displayed
+    waitFor {
```

Comment:
i am on it.
wanted to make sure that existing tests are running before writing new ones.
firefox showed the alert too slow for selenium to recognize them as displayed